app/src/main/java/com/mk/tasky/agenda/detail/reminder/presentation/DetailReminderViewModel.kt

```diff
@@ -105,20 +104,20 @@ class DetailReminderViewModel @Inject constructor(
                     isEditing = false
                 )
                 viewModelScope.launch {
-                    withContext(NonCancellable) {
-                        reminderUseCases.saveReminder(
-                            id = state.id,
-                            title = state.title,
-                            description = state.description,
-                            time = state.time,
-                            date = state.date,
-                            reminder = state.reminder
-                        )
+                    reminderUseCases.saveReminder(
+                        id = state.id,
+                        title = state.title,
+                        description = state.description,
+                        time = state.time,
+                        date = state.date,
+                        reminder = state.reminder
+                    ).onFailure {
+                        println("Couldn't save remotely") // TODO: Maybe save the ID on the DB to later save it remotely once internet is up
```

Comment:
Not just maybe and not just the ID, that's what a user expects from an offline first app 😄